ReactSkia/core_modules/RSkResourceManager.h

```diff
@@ -0,0 +1,4 @@
+//resource manager Macro.
+#define ASSETSPATH  "./assets/config/custom/assets/"
+#define EXTENTION  ".png"
+#define GENERATEPATH(URI)  ASSETSPATH+URI+EXTENTION
```

Comment:
I dont think you need to use + for concatenating in pre-processor

Reply:
here URI is string type and AssestPath and Extension is char * type. 
So, I did that. 
while pre-processing  it replace in  "./assets/config/custom/assets/" +uri+".png"
so, it not leads to error.